src/intercooler.js

```diff
@@ -1676,13 +1677,14 @@ var Intercooler = Intercooler || (function() {
       }
     }
 
-    function makeHistoryEntry(html, yOffset, url) {
+    function makeHistoryEntry(html, yOffset, url, title) {
       var restorationData = {
         "url": url,
         "id": HISTORY_SLOT_PREFIX + url,
         "content": html,
         "yOffset": yOffset,
-        "timestamp": new Date().getTime()
+        "timestamp": new Date().getTime(),
+		  "title": title
```

Comment:
Please change indentation to spaces rather than tabs for consistency.

Reply:
Fixed.